docs/rules/empty-line-between-blocks.md

```diff
@@ -29,13 +32,34 @@ When `include: false`, the following are allowed. When `include: true`, the foll
 
 ```scss
 .foo {
-  content: 'bar';
-  .baz {
-    content: 'qux';
+  content: 'foo';
+  .bar {
+    content: 'bar';
     // Waldo
-    &-- {
-      content: 'alpha';
+    &--baz {
+      content: 'baz';
     }
   }
 }
 ```
+
+### `ignore-single-line-rulesets`
+
+When `ignore-single-line-rulesets: true`, the following are allowed. When `ignore-single-line-rulesets: false`, the following are disallowed:
+
+```scss
+.foo { content: 'foo'; }
+.bar { content: 'bar'; }
+.baz { content: 'baz'; }
+```
+
+When `ignore-single-line-rulesets: false`, the following are allowed. When `ignore-single-line-rulesets: true`, the following are disallowed:
```

Comment:
These have lines between them so if true would it actually complain? when set to true? I dont think it should.. You should still be allowed to have lines between single line blocks even if you choose to ignore them here.